com.unity.render-pipelines.core/Editor/LookDev/Stage.cs

```diff
@@ -217,16 +217,28 @@ void SetGameObjectVisible(bool visible)
                 if (go == null || go.Equals(null))
                     continue;
                 foreach (UnityEngine.Renderer renderer in go.GetComponentsInChildren<UnityEngine.Renderer>())
-                    renderer.enabled = visible;
+                {
+                    if((renderer.hideFlags & HideFlags.HideInInspector) == 0 && ((renderer.hideFlags & HideFlags.NotEditable) == 0))
```

Comment:
I expect issue if someone try to display a gameobject that have an AreaLight in its hierarchy:
the emissive mesh of AreaLight is NotEditable, which means with this conditional it will not be displayed.

Reply:
It seems like it is actually displayed correctly if it is already visible, it just won't if it was set to invisble before (but I think that's correct?)

Reply:
also the mesh is not hidden in inspector, so it wouldn't trigger